src/introduction/abtnode-setup/index.md

```diff
@@ -82,7 +83,7 @@ npm install -g @abtnode/cli
 <details>
 <summary>Sample output</summary>
 
-![install ABT Node](./images/install_abtnode.gif)
+![Install ABT Node](./images/install_abtnode.gif)
```

Comment:
We should also update this section to show user how to access the node after start

Reply:
Added a section on how to access ABT Node